bears/yang/YANGBear.py

```diff
@@ -0,0 +1,44 @@
+from os.path import dirname
+
+from coala_utils.decorators import enforce_signature
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='pyang',
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r'.*:(?P<line>\d+):\s*(?P<message>.*)')
+class YANGBear:
```

Comment:
Shouldn't this class derive from some base bear class ?

Reply:
Not with `@linter` ;)